hclasses/CDebugger.cpp

```diff
@@ -49,7 +49,8 @@ void printErrorStream(const char *message) { _printErrorStreamPointer(message);
 
 void _printErrorStream(const char *pszMessage) { fprintf(stderr, "%s", pszMessage); }
 void _printWarningStream(const char *pszMessage) { fprintf(stderr, "%s", pszMessage); }
-void _printDebugStream(const char *pszMessage) { printf("%s", pszMessage); }
+// void _printDebugStream(const char *pszMessage) { printf("%s", pszMessage); }
+void _printDebugStream(const char *pszMessage) { fprintf(stderr, "%s", pszMessage); }
```

Comment:
As discussed we can keep this as it was and create a forklogger function which logs to stderr. This can be set as functionpointer in `setErrorFunction(serverErrorFunction)`, `setWarningFunction(serverWarningFunction)`, and `setDebugFunction(serverDebugFunction)`